geniza/common/metadata_export.py

```diff
@@ -78,6 +78,22 @@ def iter_export_data_as_dicts(self):
         # save
         yield from (self.get_export_data_dict(obj) for obj in iterr)
 
+    def serialize_value(self, value):
+        """A quick serialize methodpy to transform a value into a CSV-friendly string.
+
+        :param value: Any value
+        :type value: object
+        :return: Stringified value
+        :rtype: str
+        """
+        if type(value) is bool:
+            return "y" if value else "n"
+        else:
+            return str(value)
```

Comment:
this is nice, could we do the same thing for lists?

Reply:
Good idea. How about this:
```python
elif type(value) in {list,tuple,set}:
            return self.sep_within_cells.join(
                self.serialize_value(subval)
                for subval in sorted(list(value))
            )
```